tests/ios/xcodeproj/Single-Static-Framework-Project.xcodeproj/project.pbxproj

```diff
@@ -282,7 +282,7 @@
 				CLANG_ENABLE_OBJC_ARC = YES;
 				FRAMEWORK_SEARCH_PATHS = "$(PLATFORM_DIR)/Developer/Library/Frameworks";
 				GCC_PREPROCESSOR_DEFINITIONS = "$(inherited)";
-				HEADER_SEARCH_PATHS = "\"$BAZEL_WORKSPACE_ROOT/bazel-out/applebin_ios-ios_x86_64-dbg-ST-f75cadb68314/bin/tests/ios/frameworks/objc/ObjcFramework_public_hmap.hmap\" \"$BAZEL_WORKSPACE_ROOT/bazel-out/applebin_ios-ios_x86_64-dbg-ST-f75cadb68314/bin/tests/ios/frameworks/objc/ObjcFramework_private_hmap.hmap\" \"$BAZEL_WORKSPACE_ROOT/bazel-out/applebin_ios-ios_x86_64-dbg-ST-f75cadb68314/bin/tests/ios/frameworks/objc/ObjcFramework_private_angled_hmap.hmap\" \"$BAZEL_WORKSPACE_ROOT\"";
+				HEADER_SEARCH_PATHS = "\"$BAZEL_WORKSPACE_ROOT/bazel-out/applebin_ios-ios_x86_64-dbg-ST-e85ff352ea2d/bin/tests/ios/frameworks/objc/ObjcFramework_public_hmap.hmap\" \"$BAZEL_WORKSPACE_ROOT/bazel-out/applebin_ios-ios_x86_64-dbg-ST-e85ff352ea2d/bin/tests/ios/frameworks/objc/ObjcFramework_private_hmap.hmap\" \"$BAZEL_WORKSPACE_ROOT/bazel-out/applebin_ios-ios_x86_64-dbg-ST-e85ff352ea2d/bin/tests/ios/frameworks/objc/ObjcFramework_private_angled_hmap.hmap\" \"$BAZEL_WORKSPACE_ROOT/bazel-out/applebin_ios-ios_x86_64-dbg-ST-f75cadb68314/bin/tests/ios/frameworks/objc/ObjcFramework_public_hmap.hmap\" \"$BAZEL_WORKSPACE_ROOT/bazel-out/applebin_ios-ios_x86_64-dbg-ST-f75cadb68314/bin/tests/ios/frameworks/objc/ObjcFramework_private_hmap.hmap\" \"$BAZEL_WORKSPACE_ROOT/bazel-out/applebin_ios-ios_x86_64-dbg-ST-f75cadb68314/bin/tests/ios/frameworks/objc/ObjcFramework_private_angled_hmap.hmap\" \"$BAZEL_WORKSPACE_ROOT\"";
```

Comment:
This change makes sense, because ObjcFramework can be built one of two ways:

```
~/Development/rules_ios amber/use-all-search-paths bazelisk cquery 'allpaths(tests/ios/xcodeproj:Single-Static-Framework-Project, tests/ios/frameworks/objc:ObjcFramework)'
INFO: Analyzed 2 targets (0 packages loaded, 79 targets configured).
INFO: Found 2 targets...
//tests/ios/frameworks/objc:ObjcFramework (b825178b8896d4f990944aa10179e51f20dabcefac3ab234bf5d4da3df2131df)
//tests/ios/frameworks/objc:ObjcFramework (HOST)
//tests/ios/frameworks/objc:ObjcFramework (d9a500664a14607c0e7a1112f9a284574e55f557c6fc99cbbe26a2d6a972f231)
//tests/ios/xcodeproj:Single-Static-Framework-Project (756ab3ce6f028ea563926bee2bd9c47c1dbcd5b3f5aeee8d3d12d1c605038aa3)
//tests/ios/xcodeproj:Single-Static-Framework-Project (HOST)
//tests/ios/frameworks/objc:ObjcFrameworkTestLib (HOST)
//tests/ios/frameworks/objc:ObjcFrameworkTestLib_objc (HOST)
//tests/ios/frameworks/objc:ObjcFrameworkTestLib (c54ba92df8de88568a6191030eafabda830708ac76941ab9cdb7a2360ee2ea53)
//tests/ios/frameworks/objc:ObjcFrameworkTestLib_objc (c54ba92df8de88568a6191030eafabda830708ac76941ab9cdb7a2360ee2ea53)
//tests/ios/frameworks/objc:ObjcFrameworkTests_objc (3713a275cd8fa69dc716f35427b8238fe8b88e687b11e8993bece6bf66a8743d)
//tests/ios/frameworks/objc:ObjcFrameworkTests_objc (HOST)
//tests/ios/frameworks/objc:ObjcFrameworkTests.__internal__.__test_bundle (HOST)
//tests/ios/frameworks/objc:ObjcFrameworkTests.__internal__.__test_bundle.swift_runtime_linkopts (HOST)
//tests/ios/frameworks/objc:ObjcFrameworkTests.__internal__.__test_bundle (756ab3ce6f028ea563926bee2bd9c47c1dbcd5b3f5aeee8d3d12d1c605038aa3)
//tests/ios/frameworks/objc:ObjcFrameworkTests.__internal__.__test_bundle.swift_runtime_linkopts (3713a275cd8fa69dc716f35427b8238fe8b88e687b11e8993bece6bf66a8743d)
//tests/ios/frameworks/objc:ObjcFrameworkTests (756ab3ce6f028ea563926bee2bd9c47c1dbcd5b3f5aeee8d3d12d1c605038aa3)
//tests/ios/frameworks/objc:ObjcFrameworkTests (HOST)
INFO: Elapsed time: 0.239s
INFO: 0 processes.
INFO: Build completed successfully, 0 total actions
~/Development/rules_ios amber/use-all-search-paths bazelisk config b825178b8896d4f990944aa10179e51f20dabcefac3ab234bf5d4da3df2131df d9a500664a14607c0e7a1112f9a284574e55f557c6fc99cbbe26a2d6a972f231
INFO: Displaying diff between configs b825178b8896d4f990944aa10179e51f20dabcefac3ab234bf5d4da3df2131df and d9a500664a14607c0e7a1112f9a284574e55f557c6fc99cbbe26a2d6a972f231
Displaying diff between configs b825178b8896d4f990944aa10179e51f20dabcefac3ab234bf5d4da3df2131df and d9a500664a14607c0e7a1112f9a284574e55f557c6fc99cbbe26a2d6a972f231
FragmentOptions com.google.devtools.build.lib.analysis.config.CoreOptions {
  affected by starlark transition: [apple configuration distinguisher, apple_platform_type, cpu, ios_minimum_os], [apple_split_cpu, ios_minimum_os]
  transition directory name fragment: ST-e85ff352ea2d, ST-f75cadb68314
}
```